charts/coturn/values.yaml

```
@@ -41,6 +41,47 @@ tls:
 config:
   verboseLogging: false
 
+federate:
+  enabled: false
+  port: 9191
+
+  dtls:
+    enabled: true
+
+    tls:
+  #   Example:
+  #
+  #   tls:
+  #     key: "-----BEGIN EC PRIVATE KEY----- ..." # (ascii blob) private key
+  #     crt: "-----BEGIN CERTIFICATE----- ..." # (ascii blob) certificate
+  #
+  #   OR (mutually exclusive)
+  #
+  #   tls:
+  #      issuerRef:
+  #        name: letsencrypt-http01
+  #
+  #        # We can reference ClusterIssuers by changing the kind here.
+  #        # The default value is Issuer (i.e. a locally namespaced Issuer)
+  #        # kind: Issuer
+  #        kind: Issuer
+  #
+  #        # This is optional since cert-manager will default to this value however
+  #        # if you are using an external issuer, change this to that issuer group.
+  #        group: cert-manager.io
+  #
+  #      # optional labels to attach to the cert-manager Certificate
+  #      certificate:
+  #        labels: ..
+  #
+  #   host: # public hostname for which the certificate is created for.
+  #
+  # # list of host/ip/cert common names / subject alt names, and optional issuer
+  # # names to accept DTLS connections from. There can be multiple entries.
+  # remoteWhitelist:
+  #   - host: host
+  #     issuer: issuer
```

Perhaps we can be more explicit that the issuer is the common name for the issuer, e.g with this:

```suggestion
  #   - host: example.com
  #     issuer: Issuer Common Name
  #   - host: another.example.com
  #     issuer: "DigiCert SHA2 Extended Validation Server CA"
  #   - host: another-host-without-issuer.example.com
```